x-pack/plugins/endpoint/server/config.ts

```diff
@@ -15,6 +15,8 @@ export const EndpointConfigSchema = schema.object({
   enabled: schema.boolean({ defaultValue: false }),
   endpointResultListDefaultFirstPageIndex: schema.number({ defaultValue: 0 }),
   endpointResultListDefaultPageSize: schema.number({ defaultValue: 10 }),
+  resolverResultListDefaultFirstPageIndex: schema.number({ defaultValue: 0 }),
```

Comment:
Might make sense to represent `0` and `10` as constants, since they are used over and over again here.